Checking.rb

```diff
@@ -0,0 +1,43 @@
+module Bank
+  # this will inherit behavior from Account class with '<'
+  class CheckingAccount < Account
+
+    attr_reader :checks
+    def initialize(id, balance, open_date)
+      super
+      @checks_used = 0
+    end
+
+    def withdrawl(amnt_withdrawn) # add withdrawl functionality
+      fee_and_withdrawl = amnt_withdrawn + 100  # subtract $2 from balance (add to amnt_withdrawn)
+      if @balance - fee_and_withdrawl < 0  # Can't go negative.
+        # Outputs warning and un-modified balance
+        puts "You can't take out more $$ than you have ($#{@balance})! Don't forget, there is a $1 transaction fee."
+        return @balance
+      else
+      super(fee_and_withdrawl)
+      end
+      return @balance
```

Comment:
I think the `super` method should return the `@balance` variable so I don't necessarily think you need to return it here again